### PR TITLE
Configure M2 snapshot releases

### PR DESCRIPTION
**Pull request by tortmayr**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -9,15 +9,27 @@ pipeline {
...
             steps {
-                // ignore test failures since we parse the test results afterwards
-                sh 'mvn clean verify -Pp2 --batch-mode package' 
+                sh 'mvn clean verify -Pp2 -B' 
+            }
+        }
+        
+        stage ('Build: Plain Maven (M2)') {
+        	steps {
+           		sh 'mvn clean verify -Pm2 -B'
```
Comment (ndoschek) [inline Jenkinsfile:18]: Minor nitpick: Could you please align the whitespaces in the Jenkinsfile?

File: Jenkinsfile [hunk 2]
```diff
@@ -9,15 +9,27 @@ pipeline {
...
         stage('Deploy (main only)') {
             when { branch 'main' }
             steps {
-                build job: 'deploy-emfcloud-modelserver-glsp-integration-p2', wait: false
+	            parallel(
+	                p2: {
+   						build job: 'deploy-emfcloud-modelserver-glsp-integration-p2', wait: false	               
+                  	},
+                  	m2: {
+	                     build job: 'deploy-emfcloud-modelserver-glsp-integration-m2', wait: false
```
Comment (ndoschek) [inline Jenkinsfile:30]: Same as above

**Review by ndoschek — COMMENTED** [review]
Thanks a lot Tobias for adding the m2 build profile and the deployment job! Works and looks great to me!
I just noticed one minor issue: Could you please align the whitespaces in the Jenkinsfile? Thanks!

**Comment by tortmayr** [comment]
Done. Interestingly this seems to be an editor-specific issue. I edited the file with the text editor in Eclipse (with correctly aligned withtespaces) pushed the PR and the whitespaces were off. I did the same in  VS code and now everything seems to be fine :grin:

**Review by ndoschek — APPROVED** [review]
Great, thanks a lot! :tada:

**Pull request merged**